flow/packet-headers/ipv4.yaml

```diff
@@ -112,7 +112,7 @@ components:
               Source address
             format: ipv4
             default: 0.0.0.0
-            features: [count, metric_tags, auto]
+            features: [count, metric_tags, auto, random]
```

Comment:
Add one test without `random` feature and validate that `random` not auto-populate 
```
        ipv4:
          x-field-pattern:
            format: ipv4
            default: 0.0.0.0
            features: [count, auto]
```